__mocks__/getLikesMock.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export const getLikesMock = async (API_PATH) => {
```

Comment:
Mocks are essential for integration tests. So its good that you wrote a mock function.

Reply:
Thanks